python/lvmdrp/utils/metadata.py

```diff
@@ -1433,18 +1433,21 @@ def _collect_header_data(filename: str) -> dict:
                         # sci
                         'sci_ra': 'TESCIRA', 'sci_dec': 'TESCIDE', 'sci_amass': 'TESCIAM',
                         'sci_kmpos': 'TESCIKM', 'sci_focpos': 'TESCIFO',
-                        'sci_geoshadow_hgt':'GEOCORONAL SCI SHADOW_HEIGHT',
-                        'sci_moon_alt': 'SKYMODEL SCI ALT', 'sci_moon_rho': 'SKYMODEL SCI RHO',
                         # skye
                         'skye_ra': 'TESKYERA', 'skye_dec': 'TESKYEDE', 'skye_amass': 'TESKYEAM',
                         'skye_kmpos': 'TESKYEKM', 'skye_focpos': 'TESKYEFO', 'skye_name': 'SKYENAME',
-                        'skye_geoshadow_hgt':'GEOCORONAL SKYE SHADOW_HEIGHT',
-                        'skye_moon_alt': 'SKYMODEL SKYE ALT', 'skye_moon_rho': 'SKYMODEL SKYE RHO',
                         # skyw
                         'skyw_ra': 'TESKYWRA', 'skyw_dec': 'TESKYWDE', 'skyw_amass': 'TESKYWAM',
                         'skyw_kmpos': 'TESKYWKM', 'skyw_focpos': 'TESKYWFO', 'skyw_name': 'SKYWNAME',
-                        'skyw_geoshadow_hgt':'GEOCORONAL SKYW SHADOW_HEIGHT',
-                        'skyw_moon_alt': 'SKYMODEL SKYW ALT', 'skyw_moon_rho': 'SKYMODEL SKYW RHO'
+                        # sky parameters
+                        'sci_sh_hght': 'GEOCORONAL SCI_SH_HGHT', 
+                        'skye_sh_hght': 'GEOCORONAL SKYE_SH_HGHT',
+                        'skyw_sh_hght': 'GEOCORONAL SKYW_SH_HGHT',
+                        'moon_ra': 'SKYMODEL MOON_RA', 'moon_dec': 'SKYMODEL MOON_DEC',
+                        'moon_phase': 'SKYMODEL MOON_PHASE', 'moon_fli': 'SKYMODEL MOON_FLI',
+                        'sun_alt': 'SKYMODEL SUNALT', 'moon_alt': 'SKYMODEL MOONALT',
+                        'sci_moon_sep': 'SKYMODEL SCI_RHO', 'skye_moon_sep': 'SKYMODEL SKYE_RHO',
+                        'skyw_moon_sep': 'SKYMODEL SKYW_RHO'
                         }
```

Comment:
can you keep the sci, skye, and skyw parameters grouped under each section?  sky parameters that aren't prefixed or related can stay under the sky parameters group.  

Also, what happened to the moon rho and alt for each telescope?  Did we remove those from the headers?  I only see single values for those?

Reply:
Ok, I can have the tel be the primary grouping and rest under sky parameters.
The parameters '_rho' I renamed with '_sep' because I thought that would be more intuitive and there is still one for each tel.  The alt I think was a typo before and should be the moon_alt (which does not depend on tel)

Reply:
done

Reply:
thanks